File: Java/Easy/WrongAnswer/alternating-digit-sum_5.java
class Solution {
    public int alternateDigitSum(int n) {
        
        int sum = 0;
        int sign = 1;
        while (n > 0) {
            sum += sign * (n % 10);
            n = n / 10;
            sign = -sign;
        }
        return sum;
    }
}

File: Rust/Easy/WrongAnswer/alternating-digit-sum_5.rs
impl Solution {
    pub fn alternate_digit_sum(n: i32) -> i32 {
        
        let mut n = n;
        let mut sum = 0;
        let mut sign = 1;
        while n > 0 {
            let d = n % 10;
            sum += sign * d;
            sign *= -1;
            n /= 10;
        }
        sum
    }
}